index.js

```diff
@@ -1 +1,85 @@
-console.log('Hello World!');
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const chai = require('chai');
+
+const should = chai.should();
+
+/**
+ * Validates the structure of the "proof" property on a digital document.
+ *
+ * @param {object} data - A digital document with "proof" property on it.
+ * @param {string} vendor - The name of the vendor who issued the document.
+ *
+ * @returns {undefined} Just returns on success.
+ */
+function checkDataIntegrity(data, vendor) {
```

Comment:
Use options-style params:

```suggestion
function checkDataIntegrity({data, vendor}) {
```

Also, if this function doesn't actually check data integrity (which it currently doesn't ... will it ever?) then it should be renamed to match what it's actually doing. Perhaps `checkDataIntegrityProofFormat` or something like that?

Reply:
Can we assume this function will always be synchronous? Or should it return a promise just in case?

Reply:
Updated the function name to `checkDataIntegrityProofFormat` [c039eab](https://github.com/digitalbazaar/data-integrity-test-suite-assertion/pull/1/commits/c039eab3e636847943d10f0dda314fc86971bea3). 

Currently, the function doesn't contain any asynchronous operation, so it doesn't seem like we'll need to return a promise. I can't think of a use case and think we can update it in future when the need arises. but what are your thoughts on this? Should we change it to be async?

Reply:
We can leave it synchronous if it will never be used to actually check a proof itself, etc. (verify signature). If it's only ever going to check formats, it should be possible to do that synchronously.